css-brochure-form/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h2>LEARN MORE</h2>
+  <div class="format">
+    <form>
+      <div>
+        <div>
+          <label for="email">COMPANY EMAIL</label>
+        </div>
+        <div>
+          <input type="text" name="email" id="email" placeholder="sally@company.co">
+        </div>
+      </div>
```

Comment:
Whew, that is a lot of divs! This isn't necessarily a problem, but excessive use of `div` elements can make styling actually much more difficult. There's a joke we have in the developer community that overuse of divs is a disease known as `Divitis` 😂 

Try removing all but the necessary divs in your styling to try and keep your HTML more simple.